src/App.js

```diff
@@ -1,23 +1,60 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import foodsJSON from './foods.json';
+import FoodBox from './Components/FoodBox';
+import { Row, Divider, Button } from 'antd';
+import AddFoodForm from './Components/AddFoodForm';
+import Search from './Components/Search';
 
 function App() {
+  const [foods, setFoods] = useState(foodsJSON);
+  const [showFoods, setShowFoods] = useState(foodsJSON);
+
+
+    const deleteFood = (foodName) => {
+    const filteredFoods = foods.filter((food) => {
+      return food.name !== foodName; 
+    });
+    setFoods(filteredFoods);
+  };
+  
+  const addToList = (food) => {
+    const foodsCopy = [food, ...foodsJSON];
+    setFoods(foodsCopy);
+  }
+
+  const searchFoods = (query) => {
+    const filteredFoods = showFoods.filter((food) =>
+      food.name.toLowerCase().includes(query.toLowerCase())
+    );
+    setShowFoods(filteredFoods);
```

Comment:
Here, you should've called `setFoods` instead of `setShowFoods` because the initial state of `showFoods` is the same as `foods`, which is the state that is being modified when you filter the results.

```
const searchFoods = (query) => {
    const filteredFoods = showFoods.filter((food) =>
      food.name.toLowerCase().includes(query.toLowerCase())
    );
    setFoods(filteredFoods);
  };
```